site.yml

```
@@ -42,6 +42,10 @@ asciidoc:
 #   ocis
     latest-ocis-version: 'next@'   # do not change, soft set, correctly defined via antora.yml
     previous-ocis-version: 'next@' # do not change, soft set, correctly defined via antora.yml
+    # these versions are just for printing like in releases but not used for referencing
+    # needs to be changed here and not in the docs-ocis repo to be properly shown on the web
```

```suggestion
    # needs to be changed here and mandatory in the docs repo to be properly shown on the web
```
copy paste error 🤦‍♂️